cognite/client/_http_client.py

```diff
@@ -109,6 +110,8 @@ def request(self, method: str, url: str, **kwargs) -> requests.Response:
                 last_status = res.status_code
                 retry_tracker.status += 1
                 if not retry_tracker.should_retry(status_code=last_status):
+                    # Cache .json() return value in order to avoid redecoding JSON if called multiple times
+                    res.json = functools.lru_cache(maxsize=1)(res.json)
```

Comment:
why cache only one call though?

Reply:
TIL we can use decorators like that too :)

Reply:
Note that we create one lru_cache per object here, so there will only ever be one return value to store in the cache.